src/coreclr/src/jit/compiler.hpp

```diff
@@ -2729,13 +2729,31 @@ inline unsigned Compiler::fgThrowHlpBlkStkLevel(BasicBlock* block)
 */
 inline void Compiler::fgConvertBBToThrowBB(BasicBlock* block)
 {
-    // If we're converting a BBJ_CALLFINALLY block to a BBJ_THROW block,
```

Comment:
Comment not related to your change: the size of some of these "inline" functions has gotten ridiculous. Seems like we should get rid of the ".hpp" file entirely and assume a good compiler will inline appropriately.

Reply:
Agree, was going to move this one in particular, but perhaps they all should go.

Perhaps part of some broader breakup of the bigger files into related sets of functionality or something.